onchain/src/tests/hodl_quest.cairo

```diff
@@ -0,0 +1,67 @@
+use art_peace::{IArtPeaceDispatcher, IArtPeaceDispatcherTrait};
+use art_peace::quests::hodl_quest::HodlQuest::HodlQuestInitParams;
+use art_peace::quests::interfaces::{IQuestDispatcher, IQuestDispatcherTrait};
+
+use art_peace::tests::art_peace::deploy_with_quests_contract;
+use art_peace::tests::utils;
+use starknet::{ContractAddress, get_caller_address, get_contract_address, contract_address_const};
+use snforge_std as snf;
+use snforge_std::{declare, CheatTarget, ContractClassTrait};
+
+const reward_amt: u32 = 18;
+const user_extra_pixel: u32 = 0;
```

Comment:
Make user_extra_pixel 10 so that we are testing if they have enough pixels

Reply:
This was set back to 0, so nothing is being tested.

Reply:
setting this to a value keep throwing this error, 

```

Failure data:
    0x5175657374206e6f7420636c61696d61626c65 ('Quest not claimable')

```

which suggest that this does not get updated 


`art_peace_main.get_user_extra_pixels_count(user)`